src/sxastarter/package-lock.json

```diff
@@ -3563,17 +3575,17 @@
       }
     },
     "node_modules/@sitecore-feaas/clientside": {
-      "version": "0.3.10",
-      "resolved": "https://registry.npmjs.org/@sitecore-feaas/clientside/-/clientside-0.3.10.tgz",
-      "integrity": "sha512-Sww19iR64/zTyaNq1sM6NXzGOdoiYlzkEHcbwyGk0xzs6eosWVz4pHQyIZVi7XLaoiTRk56c56Qh5KCpIMXirw==",
+      "version": "0.3.16",
```

Comment:
Should we update this when we release 21.4.0?

Reply:
I wasn't sure  - that's the way it's configured in previous release, but with all the changes Components do it makes sense to lock it